contrib/autopilot/tasks.py

```diff
@@ -517,7 +533,72 @@
                              "wmt17_test/test/newstest2017-entr-ref.tr.sgm",
                              TEXT_UTF8_RAW_SGML),
                         ]),
-    # WNMT18 shared task
+    # WMT18 translation tasks
+    "wmt18_de_en": Task(description="WMT18 English--German",
```

Comment:
Super nitpick: we've been using description format like "WMT18 English-German news".  Single dash matches WMT website and we specify the news task.